src/Tables/molecules/TypeTable.tsx

```diff
@@ -42,26 +45,75 @@ const HeaderItem = styled.div<{fixedWidth?: number, alignment?: TypeCellAlignmen
 
 `;
 
+const LoadingText = styled.div`
+  color: hsla(195, 10%, 52%, 0.72);
+}
+`;
+const LoadingBox = styled.div`
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: 99;
+  background-color: hsla(210, 14%, 97%, 0.85);
```

Comment:
Can we make the `background-color` the `pureBase` in  the theme? At the moment the grey doesn't work over white but a lot of designs use the grey. We may need to update it again but if it's tied in to the theme it's a step in the right direction, even if it doesn't fix the problem just yet. I'll have to do that on my themeing part!

Reply:
ok, I will update this BG to pureBase and put the opacity on 85% outside of the backgound-color value;

Reply:
updated